src/menu/menu-section.styles.tsx

```diff
@@ -0,0 +1,28 @@
+import styled, { css } from "styled-components";
+import { Border, Colour, Spacing } from "../theme";
+import { Typography } from "../typography";
+
+// =============================================================================
+// STYLES INTERFACE
+// =============================================================================
+interface MenuSectionStyleProps {
+    $showDivider?: boolean;
+}
+
+// =============================================================================
+// MENU SECTION STYLES
+// =============================================================================
+export const Section = styled.div<MenuSectionStyleProps>`
+    ${(props) =>
+        props.$showDivider &&
+        css`
+            border-top: ${Border["width-010"]} ${Border["solid"]}
+                ${Colour["border"]};
+        `}
```

Comment:
decided to allow dividers for the section component only (and only allow the top divider), cause if menuItems/Links have dividers, they should be in another separate section for accessibility (where dividers can then be added)